docs/src/app/components/pages/components/Chip/ExampleArray.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Chip from 'material-ui/Chip';
+
+const styles = {
+  chip: {
+    margin: 4,
+  },
+  wrapper: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+};
+
+let chipData = [
+  {key: 0, label: 'Angular'},
+  {key: 1, label: 'JQuery'},
+  {key: 2, label: 'Polymer'},
+  {key: 3, label: 'ReactJS'},
+];
+
+/**
+ * An example of rendering multiple Chips from an array of values.
+ */
+export default class ChipExampleSimple extends React.Component {
+
+  constructor(props) {
```

Comment:
Well, it depends on how well stage 1 / 2 / 3 are used. Let me have a look at the npm download stats.